sunpy/map/sources/solo.py

```diff
@@ -0,0 +1,53 @@
+"""Solar Orbiter Map subclass definitions"""
+import astropy.units as u
+from astropy.coordinates import CartesianRepresentation
+from astropy.visualization import AsinhStretch, ImageNormalize
+
+from sunpy.coordinates import HeliocentricInertial
+from sunpy.map import GenericMap
+from sunpy.map.sources.source_type import source_stretch
+
+__all__ = ['EUIMap']
+
+
+class EUIMap(GenericMap):
+    """EUI Image Map
+
+    References
+    ----------
+    """
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+        self._nickname = self.detector
+        self.plot_settings['cmap'] = self._get_cmap_name()
+        self.plot_settings['norm'] = ImageNormalize(
+            stretch=source_stretch(self.meta, AsinhStretch(0.01)), clip=False)
+        self.meta['exptime'] = self.meta.get('xposure', 0.0)
```

Comment:
I am pretty sure this is also the more modern key now.

Reply:
I guess we should probably modify `exposure_time` to prioritize `'xposure'`

Reply:
This is a much larger issue that we can deal with in #4716